espnet/asr/pytorch_backend/asr.py

```diff
@@ -490,6 +494,8 @@ def train(args):
 
     # save snapshot which contains model and optimizer states
     trainer.extend(torch_snapshot(), trigger=(1, 'epoch'))
+    if args.save_interval_iters > 0:
+        trainer.extend(torch_snapshot_iter(), trigger=(args.save_interval_iters, 'iteration'))
```

Comment:
Why don't you reuse `torch_snapshot` instead of `torch_snapshot_iter`?
```suggestion
        trainer.extend(torch_snapshot(filename='snapshot.iter.{.updater.iteration}'),
                       trigger=(args.save_interval_iters, 'iteration'))
```